test/controllers/HospitalStayBeginControllerSpec.scala

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
If you agree this functionality should be moved to the HealthReasonController, then these tests can be added to `HealthReasonControllerSpec`